package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vue-cli-plugin-single-spa",
   "description": "A vue-cli plugin for single-spa applications and parcels",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
Please revert, as bumping new versions is done during publish, not in each pull request.

```suggestion
```